### PR TITLE
bpo-31853: Replaced socket.method calls with super() in SSLSocket.

### PR DESCRIPTION
**Pull request by atombrella**
https://bugs.python.org/issue31853

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

**Comment by atombrella** [comment]
It's an issue many places in different classes, e.g., also in `PyZipFile` so how should the issue be created?

**Comment by atombrella** [comment]
@tiran Is there anything stalling this?

File: Lib/ssl.py [hunk 1]
```diff
@@ -781,17 +781,16 @@ def __init__(self, sock=None, keyfile=None, certfile=None,
         self.do_handshake_on_connect = do_handshake_on_connect
         self.suppress_ragged_eofs = suppress_ragged_eofs
         if sock is not None:
-            socket.__init__(self,
-                            family=sock.family,
+            super().__init__(family=sock.family,
```
Comment (tiran) [inline Lib/ssl.py]: Please fix indention

**Review by tiran — CHANGES_REQUESTED** [review]
Sorry, I forgot about your patch. LGTM, please fix indention and I'll merge it later today.

File: Lib/ssl.py (hunk 1 above)
Comment (atombrella) [reply Lib/ssl.py]: Should be done. One attribute per line seems to be the style.

**Review by tiran — APPROVED** [review]

**Pull request merged**